src/main/java/our/yurivongella/instagramclone/controller/PostController.java

```diff
@@ -26,14 +28,17 @@
 
     @GetMapping("/{postId}")
     public ResponseEntity<PostReadResponseDto> read(@PathVariable Long postId) {
-        PostReadResponseDto postResponseDto = postService.read(postId);
-        return ResponseEntity.ok(postResponseDto);
+        return ResponseEntity.ok(postService.read(postId));
     }
 
-
     @DeleteMapping("/{postId}")
     public ResponseEntity<String> delete(@PathVariable Long postId) {
         Long id = postService.delete(postId);
         return ResponseEntity.ok(id + "인 포스트가 삭제되었습니다.");
     }
+
+    @GetMapping("/postlist/{userId}")
```

Comment:
RESTful 형식에 맞게 네이밍 해주세요
`Controller` 에 상단에 있는 `@RequestMapping("/post")` 를 제거가 필요합니다.

```suggestion
    @GetMapping("/members/{memberId}/posts")
```

Reply:
반영하여 수정하였습니다.
https://github.com/INSPJT/instagram_clone_backend/pull/35/commits/40ec8ced7f77d2081eec325886ecbdd5ceca8a89